operators/postgresql/4.7.5/manifests/postgresoperator.v4.7.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,959 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: 'postgresoperator.v4.7.5'
+  annotations:
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.8"}]'
```

Comment:
For kubernetes one should add `operatorhub.io/ui-metadata-max-k8s-version: "1.21"`

Reply:
Thank you @mvalarh,  I updated with the correct annotation.  Would you be able to point me in the direction of the documentation for that annotation?  I used the olm properties by mistake when following this document https://k8s-operatorhub.github.io/community-operators/packaging-required-criteria-ocp/

Thanks Again for the help!